File: app/src/main/java/dev/ragnarok/fenrir/db/column/FavePageColumns.java
package dev.ragnarok.fenrir.db.column;

import android.provider.BaseColumns;

public final class FavePageColumns implements BaseColumns {
    public static final String TABLENAME = "fave_pages";
    public static final String GROUPSTABLENAME = "fave_groups";

    public static final String FULL_ID = TABLENAME + "." + _ID;

    public static final String FULL_GROUPS_ID = GROUPSTABLENAME + "." + _ID;

    public static final String DESCRIPTION = "description";
    public static final String UPDATED_TIME = "updated_time";
    public static final String FAVE_TYPE = "fave_type";

    private FavePageColumns() {
    }

}

File: app_fenrir/src/main/kotlin/dev/ragnarok/fenrir/db/column/FavePageColumns.kt
package dev.ragnarok.fenrir.db.column

import android.provider.BaseColumns

object FavePageColumns : BaseColumns {
    const val TABLENAME = "fave_pages"
    const val GROUPSTABLENAME = "fave_groups"
    const val FULL_ID = TABLENAME + "." + BaseColumns._ID
    const val FULL_GROUPS_ID = GROUPSTABLENAME + "." + BaseColumns._ID
    const val DESCRIPTION = "description"
    const val UPDATED_TIME = "updated_time"
    const val FAVE_TYPE = "fave_type"
}